CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Change the `otlpmetric.Client` interface's `UploadMetrics` method to accept a single `ResourceMetrics` instead of a slice of them. (#2491)
 - Specify explicit buckets in Prometheus example. (#2493)
 
+### Removed
+- Updated to /go.opentelemetry.io/proto/otlp@v0.12.0. This version removes a number of deprecated methods. (#2546)
```

Comment:
```suggestion

- Updated `go.opentelemetry.io/proto/otlp` from `v0.11.0` to `v0.12.0`. This version removes a number of deprecated methods. (#2546)
```

I'm not 100% sure this is needed, it isn't going to be a user facing change, but probably fine being overly inclusive in the changelog.